src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -74,7 +75,15 @@ uiRoutes
   }
 });
 
-app.controller('discover', function ($scope, config, courier, $route, $window, Notifier,
+app
+  .directive('discoverApp', function () {
```

Comment:
Indent issue, and you can keep the `.directive` part on the same line if you prefer.